crypto/edhoc-crypto-rustcrypto/Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
 name = "edhoc-crypto-rustcrypto"
```

Comment:
@chrysn feel free to add yourself as author of this and the crypto-trait crates.

Reply:
Done (wasn't sure we're doing `authors = `, cargo doesn't set it by default any more); crypto-trait has been taken up into -consts.